tests/Integration/ResourceStatusSystem/Test_ResourceManagement.py

```diff
@@ -27,15 +30,15 @@ def test_addAndRemove():
   # ...............................................................................
 
   res = rsClient.addOrModifyAccountingCache('TestName12345', 'plotType', 'plotName', 'result', dateEffective, lastCheckTime)
-  assert res['OK'] == True
+  assert res['OK'] is True
```

Comment:
You mean this? https://docs.python.org/2/reference/simple_stmts.html#the-assert-statement
At the same time I don't find in the code the "-O" (or "-OO"), where is it? @atsareg